doc/manual/src/advanced-topics/post-build-hook.md

```diff
@@ -17,9 +17,8 @@ the build loop.
 
 # Prerequisites
 
-This tutorial assumes you have [configured an S3-compatible binary
-cache](../package-management/s3-substituter.md), and that the `root`
-user's default AWS profile can upload to the bucket.
+This tutorial assumes you have configured an [S3-compatible binary cache](../command-ref/conf-file.md#conf-substituters),
```

Comment:
```suggestion
This tutorial assumes you have configured an [S3-compatible binary cache](@docroot@/command-ref/new-cli/nix3-help-stores.md#s3-binary-cache-store) as a [substituter](../command-ref/conf-file.md#conf-substituters),
```

Reply:
I think we want to put back something like 

> and that the `root` user's default AWS profile can upload to the bucket.

Because it very possible to use `s3://` without write acess